app/Enums/ClassType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+enum ClassType: string
+{
+    case LABORATORY = "laboratory";
```

Comment:
Why UPPERCASED?

Reply:
I had uppercased enums in other projects too.

Reply:
PHP documentation uses PascalCased cases. I would use that casing in all our projects.

Reply:
I will stay with that

Reply:
https://www.php-fig.org/per/coding-style/#9-enumerations

Reply:
changed to PascalCase